Assets/HoloToolkit/Input/Scripts/Utilities/Managers/MixedRealityTeleport.cs

```diff
@@ -287,11 +290,18 @@ public void DoStrafe(Vector3 strafeAmount)
         /// <param name="worldPosition"></param>
         public void SetWorldPosition(Vector3 worldPosition)
         {
+            var orginaly = transform.position.y;
```

Comment:
typo: `orginal` -> `original`.
Also, wouldn't hurt to capitalize the `Y` as well, to make it a bit clearer what this variable is and harder to confuse with the word `originally` or something.